### PR TITLE
Fix common carp homepage

### PR DESCRIPTION
**Pull request by ndliberial**
Found out the html files for Common carp are missing on the website, hence this resubmission.

### CODE REVIEW

File: ensembl/htdocs/ssi/species/Cyprinus_carpio_carpio_annotation.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+</p>
```
Comment (jgtate) [inline ensembl/htdocs/ssi/species/Cyprinus_carpio_carpio_annotation.html:1]: Could you remove this stray "close paragraph" tag?

**Review by jgtate — CHANGES_REQUESTED** [review]
There's one stray close tag but other than that it looks fine. I'll merge it and push it out as soon as that's fixed.

File: ensembl/htdocs/ssi/species/Cyprinus_carpio_carpio_annotation.html (hunk 1 above)
Comment (ens-st3) [reply ensembl/htdocs/ssi/species/Cyprinus_carpio_carpio_annotation.html:1]: We shouldn't have all these merge commits I think ?

**Review by ndliberial — COMMENTED** [review]
Tag removed

**Comment by jyothishnt** [comment]
We cannot merge this PR. There are 2 options I think it would work:
1. Create a new branch from `postreleasefix/106` branch and cherrypick the necessary commits from this branch and create a new PR
2. We cherrypick the necessary commits into `postreleasefix/106` and do an mpush

**Pull request closed without merging**